spectrocloud/resource_registry_oci_ecr.go

```diff
@@ -129,6 +130,15 @@ func resourceRegistryEcrRead(ctx context.Context, d *schema.ResourceData, m inte
 		if err := d.Set("credentials", credentials); err != nil {
 			return diag.FromErr(err)
 		}
+	} else if registry.Spec.Credentials.CredentialType == models.V1AwsCloudAccountCredentialTypeSecret {
```

Comment:
can we use here switch statement please @SivaanandM would be more clear.

Reply:
changed to switch case